src/main/java/com/PhotomosaicSara/image/Main.java

```diff
@@ -0,0 +1,11 @@
+package com.PhotomosaicSara.image;
+
+import java.io.IOException;
+
+public class Main {
+
+    public static void main(String[] args) throws IOException {
+        new Photomosaic(new Service()).process();
```

Comment:
In the perfect world this method should work like a cake recipe. You should have classes responsible for multiple aspects of processing and orchestration part set here. Example:
<img width="711" alt="Screenshot 2023-02-05 at 19 02 56" src="https://user-images.githubusercontent.com/7178722/216836810-69a5b2d8-c897-407d-9a0f-1676d7617d23.png">